### DISCUSSION TITLE
PRG file with Sidekick (Pi3 A+)

### DISCUSSION THREAD

**Discussion by repetto74** [frntc/Sidekick64#88] [General]
Hi,

Although CRT files are all loading fine, I cannot manage to load any PRG file with the Sidekick. Each time I'll just get a freezed C64 prompt (with a @ showing up on top of the screen).
Did I miss something with settings?
PRG have been copied in the PRG folder on the SD card.

**Comment by repetto74** [comment]
Found that PRG must be crunched to work. I was loading uncrunched PRG files!!

**Comment by blindguy** [comment]
Awesome!!! How do you 'crunch' a prg file ??

**Comment by repetto74** [comment]
I do not have a clue on how to do that. Crunched PRG files are already
available in the OneLoad64 collection for the Kung Fu Flash.

On Mon, Aug 28, 2023 at 8:56 PM blindguy ***@***.***> wrote:

> Awesome!!! How do you 'crunch' a prg file ??
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/frntc/Sidekick64/discussions/88#discussioncomment-6845789>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQTKN324JORWFJW4CQLIRQDXXTSVVANCNFSM6AAAAAA4BWBVVA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>


-- 
Riccardo Repetto
@ AppleWorld...

**Reply by blindguy** [reply]
Bummer, I dont use that cart.  Thanks for the info, I'l try to do some reading.
